playbooks/adhoc/openshift_hosted_logging_efk.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 - hosts: masters[0]
   roles:
-  - role: openshift_hosted_logging
+  - role: openshift_logging
```

Comment:
The intention is correct, though again, I'm not sure why we have so many playbooks for logging